components/app_bar/app_bar.tsx

```diff
@@ -73,6 +77,10 @@ export default function AppBar() {
         return null;
     }
 
+    if (isAdmin) {
```

Comment:
Thanks for the input!
Requirements on the ticket specified only system admins, and reviewing other places where marketplace is accessed are checking `SYSCONSOLE_READ_PLUGINS`.
At least I think we need to make sure it's enabled on the server, will work on adding it.

And maybe @esethna can confirm if should include non system admin users with either `SYSCONSOLE READ_PLUGINS` or `SYSCONSOLE_WRITE_PLUGINS`.

Reply:
Year FIY I took this info from: https://github.com/mattermost/mattermost-webapp/blob/master/components/global_header/left_controls/product_menu/product_menu_list/product_menu_list.tsx#L207-L224

This is the condition we use to show or not the marketplace in the "product" menu

Reply:
+1 - we should use the permissions systems given that if there are custom permission schemes, they may grant plugin write access to users that may not be admins. Admins have the `Permissions.SYSCONSOLE_WRITE_PLUGINS]` so that should be enough. 

As Julian pointed out, the action menu does just check for `isAdmin` to show or not that marketplace entry. We should fix that. Are you open to include that as part of this PR?

For now we don't allow to open the marketplace if they have only `SYSCONSOLE_READ_PLUGINS` access or regular users. This is something we could introduce in the future with the functionality of Notify Admin to install plugins. cc/ @esethna

Reply:
@marianunez Sure, will add the privilege check in the action menu.
I see that the command `/marketplace` is checking for the read plugins privilege (`SYSCONSOLE_READ_PLUGINS`) https://github.com/mattermost/mattermost-webapp/blob/master/actions/command.ts#L131-L133
Should this be changed as well to check the write one instead?

Reply:
@marianunez Updated to check for both Marketplace enabled and `SYSCONSOLE_WRITE_PLUGINS` permission in:
- Channel header
- App bar
- Post actions menu

Pending confirmation on the `/marketplace` command.